src/components/dashboard/Claim.js

```diff
@@ -46,20 +57,34 @@ const Claim = props => {
   const [showDialog, hideDialog, showErrorDialog] = useDialog()
   const [loading, setLoading] = useState(false)
   const [claimInterval, setClaimInterval] = useState(null)
-  const [state, setState]: [ClaimState, Function] = useState({
+  const [pageState, setPageState]: [ClaimState, Function] = useState({
```

Comment:
why not claimState (according to the Flow's type name) ?

Reply:
ok, let it be claimState)